### ISSUE TITLE
Missing AttributeValueTranslation.name for richText attributes

### ISSUE THREAD

**Issue by fowczarek** [saleor/saleor#10924]
For reich text attributes, Saleor sets `AttributeValue.name` to a text summary of the `richText` field. We should implement the same behavior for `AttributeValueTranslation.name`.

In addition, the existing `AttributeValueTranslation` values should be updated through a migration.

**Comment by fowczarek** [comment]
To check it we should create a Product with an assigned RichText Attribute value. 
Next, AttributeValue should be translated to any language and verified with the query bellow. 
`product.attributes.values.translation.name` should have first 99 chars from `richText` field without formation. 
Verification query:
```graphql
{
  product(id: "<ProductID>", channel: "<channel-slug>") {
    name
    attributes {
      values {
        name
        richText
        translation(languageCode: PL) {
          name
          richText
        }
      }
    }
  }
}
```

**Comment by fowczarek** [comment]
Form 3.5, Saleor contains a similar bug for PlainText attributes. Also will be fixed by me in #10986.

**Comment by fowczarek** [comment]
Fix released in:
3.1.29
3.2.19
3.3.31
3.4.30
3.5.31
3.6.26
3.7.26
